tests/integration/receive.spec.js

```diff
@@ -164,4 +195,102 @@ describe('Liquality wallet- Receive-["mainnet"]', async () => {
     // After done
     await overviewPage.CheckAssertOverviewDetails(page, nearPlatform)
   })
+  it('Check Receive for ARBETH', async () => {
```

Comment:
Looks like a lot of duplication here. Is it necessary? Could it use a generic function where you pass the asset and it will return the test?

Reply:
Yes was thinking about that, will refactor that

Reply:
Remove those duplicated code and added generic function